src/renderer/components/GameTeam/QuessYearQuestionTeam/GuessYearQuestionTeam.tsx

```diff
@@ -40,26 +43,38 @@ function GuessYearQuestionTeam() {
         <div className={styles.questionContent}>
           <span>{question.questionText}</span>
           <img src={question.imageUrl!}></img>
-          <span>Select the year</span>
+          {!submitted ? <span>Select the year</span> : <span></span>}
+          {showSolution && <span>Winner: {teamAnswerResults?.find((x) => x.correct)?.teamName}</span>}
         </div>
-        <span className={styles.values}>
-          <span className={styles.value}>{winningScore}</span>
-          <Slider
-            onChange={handleWinningScoreChange}
-            value={winningScore}
-            aria-label="Winning score"
-            valueLabelDisplay="off"
-            step={1}
-            marks
-            min={1900}
-            max={2024}
+        {teamAnswerResults?.length ? (
+          <MultiSliderView
+            min={1800}
```

Comment:
extract these constants and define at the start of the component

Reply:
done